src/main/java/net/sf/jabref/event/location/EntryEventLocation.java

```diff
@@ -0,0 +1,25 @@
+/*  Copyright (C) 2003-2016 JabRef contributors.
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License along
+    with this program; if not, write to the Free Software Foundation, Inc.,
+    51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+*/
+package net.sf.jabref.event.location;
+
+/**
+ * This enum entries represent the radius which is affected by posting different EntryEvents.
+ */
+public enum EntryEventLocation {
```

Comment:
Mhh...scope also implies that this event is only valid in some specific area, like you describe in the comment "the affected radius". 
But the event does not know if a listener should actually do something or not. It just knows where it comes from (source / origin / ?).
Example: an entry is added remotely (source = remote). The remote sync does not want to handle the event (scope = local).